inclusion_connect/middleware.py

```diff
@@ -9,3 +11,19 @@ def middleware(request):
         return response
 
     return middleware
+
+
+def limit_staff_users_to_admin(get_response):
+    def middleware(request):
+        user = request.user
+
+        if user.is_staff and not request.path.startswith("/admin/") and not request.path == "/403/":
+            exception = mark_safe(
+                "Les comptes administrateurs n'ont pas accès à cette page.<br>"
+                "<a href=\"{% url 'admin:logout' %}\">Vous pouvez-vous déconnecter ici.</a>"
+            )
```

Comment:
On est en Python, pas besoin d’utiliser le moteur de templating HTML.
```suggestion
            exception = format_html(
                "Les comptes administrateurs n'ont pas accès à cette page.<br>"
                '<a href="{}">Vous pouvez-vous déconnecter ici.</a>',
                reverse("admin:logout"),
            )
```